blogposts/2021/role-of-observability-high-performing-teams.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Monitoring is no longer enough: For high-performing development teams you need observability tools"
+author: Kevin Graham
+authorUrl: 
+description: "Observability and tracking the right metrics are critical to development teams' success. We chatted to Charity Majors of Honeycomb about bridging the gap between dev and ops, and making software more comprehensible to everyone."
+publishDate: 2021-06-04T18:00-07:00
+tags: [blog]
+slug: role-of-observability-tools-high-performing-development-teams
+heroImage: https://sourcegraphstatic.com/blog/high-performing-development-teams-observability.gif
+socialImage: https://sourcegraphstatic.com/blog/high-performing-development-teams-observability.gif
+published: true
+---
+
+![Facilitating high-performing teams](https://sourcegraphstatic.com/blog/high-performing-development-teams-observability.gif)
+
+Legacy methods for predicting system failures are no longer enough. To bring development and operation teams together more effectively, you need modern observability and monitoring tools designed for highly distributed systems and microservices architectures.
```

Comment:
I think it would be helpful to give some high-level context or a clue as to why legacy methods are no longer enough here.